src/Compilers/CSharp/Test/Syntax/Parsing/RoundTrippingTests.cs

```diff
@@ -843,7 +843,7 @@ class AAttribute : Attribute {
 class C {
 }
 ";
-            ParseAndRoundTripping(text, -1);
+            ParseAndRoundTripping(text);
```

Comment:
>ParseAndRoundTripping(text); [](start = 12, length = 28)

It looks like we previously expected some errors. What were they and why are they gone?